test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1070,6 +1070,24 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion charset encoding tests
 
+    It "Verifies Invoke-WebRequest includes Content headers in Headers property" {
+        $command = "Invoke-WebRequest -Uri 'http://httpbin.org/get'"
```

Comment:
Use HTTPListener here instead of http://httpbin.org/get.  
The tests under Context "BasicHtmlWebResponseObject Encoding tests" illustrate using 'test=response&output=$otuput' to pass HTML you want to get in the response.  You should consider extending httplistener's 'response' option to allow additional options, such as specifying a content-type header value or other headers to support your testing.  let me know if you need any help with it.